google-cloud-examples/src/main/java/com/google/cloud/examples/pubsub/snippets/CreateSubscriptionAndConsumeMessages.java

```diff
@@ -31,7 +30,7 @@
  * A snippet for Google Cloud Pub/Sub showing how to create a Pub/Sub pull subscription and
  * asynchronously pull messages from it.
  */
-public class CreateSubscriptionAndPullMessages {
+public class CreateSubscriptionAndConsumeMessages {
```

Comment:
Yes the readme is updated in this commit https://github.com/GoogleCloudPlatform/google-cloud-java/pull/1899/commits/6d6bff6158a3c746481813b0d8f2000021a785fd